src/model/operator.ts

```diff
@@ -5,7 +5,18 @@ export type Operator = {
   id: string
 }
 
-export const getRelevantOperators = async () => {
+export const getRelevantOperators = async (onlyRelevantOperators: boolean) => {
+  const agencyList = await getAgencyList()
+  if (!onlyRelevantOperators) {
+    const allOperators = Array.from(
+      agencyList.map((agency) => ({
+        name: agency.agency_name,
+        id: agency.operator_ref.toString(),
+      })),
+    )
+    allOperators.push({ name: 'הכל', id: '' })
```

Comment:
why do we need the `הכל` here?
```suggestion
    const allOperators = agencyList.map((agency) => ({
        name: agency.agency_name,
        id: agency.operator_ref.toString(),
      }))
    allOperators.push({ name: 'הכל', id: '' })
```

Reply:
Changed, Thanks! (assumed the problem with adding this in this function doesn't fit the function name?)

Reply:
Thanks! It's also about the data model should not be aware of the UI options